addons/pos_sale/static/tests/tours/pos_sale_tour.js

```diff
@@ -178,3 +178,17 @@ registry.category("web_tour.tours").add("PosOrderDoesNotRemainInList", {
             PosSale.checkOrdersListEmpty(),
         ].flat(),
 });
+
+registry.category("web_tour.tours").add("PosSettleCustomPrice", {
+    test: true,
+    url: "/pos/ui",
+    steps: () =>
+        [
+            Dialog.confirm("Open session"),
+            PosSale.settleNthOrder(1),
+            ProductScreen.selectedOrderlineHas("product_a", "1", "100"),
```

Comment:
Isn't it "Product A" ?

Reply:
But I can create a new product "Product A" instead of using `product_a` from the class. Up to you

Reply:
No that is ok if it is product_a but I could not find the place where it was created as 'product_a'. I guess I'm tired today ahah